test/acceptance/features/steps/steps.py

```diff
@@ -284,6 +279,53 @@ def then_envFrom_contains_intermediate_secret_name(context, intermediate_secret_
         intermediate_secret_name, wait=True, timeout=120) is not None, f"There is no deployment with intermediate secret {intermediate_secret_name}"
 
 
+# STEP
+@given(u'OLM Operator Backend is running')
```

Comment:
please make `Backed` passed as argument and used for locating the proper yaml file. with that we will be able to play with different operator manifest files.